tap_chargebee/schemas/orders.json

```diff
@@ -0,0 +1,393 @@
+{
+	"type": ["null", "object"],
+	"properties": {
+		"id": {
+			"type": ["null", ""]
```

Comment:
I also get an error here as this needs to have a concrete JSON Schema type like `"string"`